Mathlib/FieldTheory/Differential/Basic.lean

```diff
@@ -89,4 +87,87 @@ lemma _root_.algebraMap.coe_logDeriv {F K : Type*} [Field F] [Field K] [Differen
     [Differential K] [Algebra F K] [DifferentialAlgebra F K]
     (a : F) : logDeriv a = logDeriv (a : K) := (logDeriv_algebraMap a).symm
 
+/--
+The differential of a separable value in a field extension.
+-/
+noncomputable def implicitDeriv' (F : Type*) {K : Type*} [CommRing F] [Differential F] [Field K]
+    [Algebra F K] (x : K) : K :=
+  - (aeval x (mapCoeffs (minpoly F x))) / (aeval x (derivative (minpoly F x)))
+
+variable {F : Type*} [Field F] [Differential F] [CharZero F]
+
+noncomputable instance (p : F[X]) [Fact (Irreducible p)] [Fact p.Monic] :
```

Comment:
Maybe it's simpler to expand the `implicitDeriv'` in the definition of that instance.

Reply:
Can you add the lemma that says that this is the unique structure of a differential algebra on `AdjoinRoot p` (provided the separability assumption holds true)?